### PR TITLE
lowdown: upgrade to 0.3.2

### PR DESCRIPTION
**Pull request by janstary**
* brings its own compacts.c, no need for snowleopard_fixes
* drop the corresponding patches
* tweak platforms

Tested on 10.13.4 and 10.6.8

### CODE REVIEW

File: textproc/lowdown/Portfile [hunk 1]
```diff
@@ -1,10 +1,8 @@
 PortSystem		1.0
-PortGroup		snowleopard_fixes 1.0
 
 name			lowdown
 categories		textproc
-version			0.3.1
-revision		1
+version			0.3.2
 license			BSD
```
Comment (pmetzger) [inline textproc/lowdown/Portfile]: Indentation on "version" and "license" is wrong, and should be fixed. Are there tabs in use here? If so, see: https://guide.macports.org/#development.practices.portstyle — tabs violate the style guide.

**Comment by janstary** [comment]
tweaked and squashed

File: textproc/lowdown/Portfile [hunk 2]
```diff
@@ -1,26 +1,22 @@
...
 
-description		simple markdown translator
-long_description	lowdown translates markdown to either HTML	\
-			or a roff document using either -man or -ms
-homepage		http://kristaps.bsd.lv/lowdown/
-master_sites		http://kristaps.bsd.lv/lowdown/snapshots/
-maintainers		{stare.cz:hans @janstary}
-platforms		openbsd darwin linux
+description             simple markdown translator
+long_description        lowdown translates markdown to either HTML	\
```
Comment (pmetzger) [inline textproc/lowdown/Portfile]: FYI, there's still a tab at the end of that line...

Comment (pmetzger) [reply textproc/lowdown/Portfile]: (Thanks for cleaning this up!)

Comment (janstary) [reply textproc/lowdown/Portfile]: eye-ripping tab removed, squashed.

**Comment by pmetzger** [comment]
Merged. Thank you, @janstary!

**Pull request merged**